src/AppBundle/Controller/SecurityController.php

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
In fact, does this even work? Since the controller is a service (and I think? that the routing automatically knows to use the service, since it exists), the container is never set, correct? So `$this->render()` is broken.